icu4c/source/tools/icu-svnprops-check.py

```diff
@@ -158,32 +84,28 @@ def main(argv):
         usage()
         sys.exit()
 
-    parse_auto_props()
-    output = runCommand("svn ls -R ");
+    output = runCommand("git ls-files ");
     file_list = output.splitlines()
 
+    source_file_re = re.compile(".*((?:\\.c$)|(?:\\.cpp$)|(?:\\.h$)|(?:\\.java$))")
+    
     for f in file_list:
+        print "Processing file " + f
```

Comment:
Isn't this going to produce a lot of output, very quickly?
In the old days, checking the svn properties on each file was very slow, so seeing some output could be helpful to know that the script hadn't gotten stuck.